lib/modules/datasource/galaxy-collection/readme.md

```diff
@@ -0,0 +1,28 @@
+By default, the `galaxy-collection` datasource checks for dependencies on `https://galaxy.ansible.com`.
+
+But you can set your own registries.
+You can set a `source` in your `requirements.yaml` file, _or_ write a `packageRule` to set a new `registryURLs`.
+This allows you to use Renovate with a private automation hub.
```

Comment:
```suggestion
By default, the `galaxy-collection` datasource checks for dependencies on `https://galaxy.ansible.com`.
But you can override the default if you want.

Set your own registries by:

- setting a `source` in your `requirements.yaml` file, _or_
- writing a `packageRule` to set a new `registryURLs`

Then you can use Renovate with a private automation hub.
```

How about this? I think a list is easier to understand. I've also moved the sentences about the default and overriding into a single paragraph. Plus rewritten some sentences.